src/apm.ts

```diff
@@ -1,13 +1,14 @@
-import apm from 'elastic-apm-node';
+import { Apm } from '@frmscoe/frms-coe-lib/lib/services/apm';
 import { configuration } from './config';
 
-if (configuration.apm.active === 'true') {
-  apm.start({
-    serviceName: configuration.apm?.serviceName,
-    secretToken: configuration.apm?.secretToken,
-    serverUrl: configuration.apm?.url,
-    usePathAsTransactionName: true,
-    active: Boolean(configuration.apm?.active),
-    transactionIgnoreUrls: ['/health'],
-  });
-}
+const { serviceName, secretToken, url, active } = configuration.apm;
```

Comment:
worth declaring these new vars for one-time-use?

Reply:
Not at all: fixed with https://github.com/frmscoe/tms-service/pull/49/commits/0ff09614ee4470966b75b53d215cb148fd1dfb2d